cli/QuickstepCli.cpp

```diff
@@ -262,14 +262,19 @@ int main(int argc, char* argv[]) {
       DefaultsConfigurator::GetNumNUMANodesCoveredByWorkers(worker_cpu_affinities);
 
   if (quickstep::FLAGS_preload_buffer_pool) {
+    std::chrono::time_point<std::chrono::steady_clock> preLoadStart, preLoadEnd;
+    preLoadStart = std::chrono::steady_clock::now();
+    printf("Preloading the buffer pool ... ");
     quickstep::PreloaderThread preloader(*query_processor->getDefaultDatabase(),
                                          query_processor->getStorageManager(),
                                          worker_cpu_affinities.front());
-    printf("Preloading buffer pool... ");
+
     fflush(stdout);
```

Comment:
You may want to move this line, along with the `printf("Preloading ...")`, so that the output will be printed immediately.